frontend/src/lib/components/DefinitionPopover/DefinitionPopoverContents.tsx

```diff
@@ -196,7 +195,7 @@ function DefinitionView({ group }: { group: TaxonomicFilterGroup }): JSX.Element
                 <DefinitionPopover.Section>
                     <DefinitionPopover.Card
                         title="Sent as"
-                        value={<span style={{ fontFamily: 'monaco', fontSize: 12 }}>{_definition.name}</span>}
```

Comment:
This was Times New Roman on all systems other than macOS, because Monaco is an Apple font LOL. Thanks UI snapshots!